README.md

```diff
@@ -232,6 +232,14 @@ Note that you'll be responsible for updating manually.
 - `libclang` (see more info at [https://crates.io/crates/clang-sys](https://crates.io/crates/clang-sys))
 - `libsqlite3`
 
+### Cargo build feature for SELinux
+
+If your target system uses SELinux, you will want to enable the
+'selinux' feature when building the innernet binary. This will ensure
+that innernet maintains the correct selinux context on the /etc/hosts
+file when adding hosts.  To do so add ```--features selinux``` to the
+```cargo build``` options.
+
 Build:
 
 ```sh
```

Comment:
OK, here's another patch with those changes.  I also remembered that the build host will need the selinux headers from libselinux-devel and added a comment to that effect.

```

From cb9d0735ab3681bc46fe902a770753be3b3aec81 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=BCrgen=20Botz?= <jurgen@botz.org>
Date: Tue, 7 May 2024 06:41:06 +0200
Subject: [PATCH] Reformatted & repositioned and improved doc about selinux
 feature

---
 README.md | 14 ++++++--------
 1 file changed, 6 insertions(+), 8 deletions(-)

diff --git a/README.md b/README.md
index aa9cec3..272c978 100644
--- a/README.md
+++ b/README.md
@@ -226,20 +226,18 @@ Note that you'll be responsible for updating manually.
 
 ## Development
 
+### Cargo build feature for SELinux
+
+If your target system uses SELinux, you will want to enable the 'selinux' feature when building the innernet binary.
+This will ensure that innernet maintains the correct selinux context on the /etc/hosts file when adding hosts.  To do so add ```--features selinux``` to the ```cargo build``` options.
+The `selinux-devel` package will need to be installed for the correct headers.
+
 ### `innernet-server` Build dependencies
 
 - `rustc` / `cargo` (version 1.50.0 or higher)
 - `libclang` (see more info at [https://crates.io/crates/clang-sys](https://crates.io/crates/clang-sys))
 - `libsqlite3`
 
-### Cargo build feature for SELinux
-
-If your target system uses SELinux, you will want to enable the
-'selinux' feature when building the innernet binary. This will ensure
-that innernet maintains the correct selinux context on the /etc/hosts
-file when adding hosts.  To do so add ```--features selinux``` to the
-```cargo build``` options.
-
 Build:
 
 ```sh
-- 
2.45.0
```

Reply:
Thanks @jebotz, I applied the patch. Sorry for the long wait on this one!